.circleci/config.yml

```diff
@@ -114,24 +114,39 @@ jobs:
           name: Push docker images
           command: |
             echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
-            docker tag \
-              scalableminds/webknossos:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} \
-              scalableminds/webknossos:${CIRCLE_BRANCH}
-            docker push scalableminds/webknossos:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
-            docker push scalableminds/webknossos:${CIRCLE_BRANCH}
-            docker tag \
-              scalableminds/webknossos-datastore:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} \
-              scalableminds/webknossos-datastore:${CIRCLE_BRANCH}
-            docker push scalableminds/webknossos-datastore:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
-            docker push scalableminds/webknossos-datastore:${CIRCLE_BRANCH}
-            docker tag \
-              scalableminds/webknossos-dev \
-              scalableminds/webknossos-dev:$CIRCLE_BRANCH
-            docker push scalableminds/webknossos-dev:$CIRCLE_BRANCH
+            if [ -n "$CIRCLE_TAG" ]; then
```

Comment:
Can you briefly explain how this branch is hit?

Reply:
CircleCI has separate builds for tags, even if they come from the same commit.

Reply:
`CIRCLE_BRANCH` is set for the commit/branch build, and `CIRCLE_TAG` is set for the tag build, which are separate things in CircleCI.